api/migrations/2020-11-08-174607_trigger_users/up.sql

```diff
@@ -0,0 +1,35 @@
+CREATE OR REPLACE FUNCTION delete_dangling_program_records()
+    RETURNS trigger AS
+$$
+BEGIN
+    WITH pbs as (
+        SELECT min(cpu_cycles) as cpu_cycles, min(instructions) as instructions, min(registers_used) as registers_used, min(stacks_used) as stacks_used
+        FROM "user_program_records" as upr
+        WHERE upr.user_id = NEW.user_id 
```

Comment:
So is `NEW` defined implicitly for insert triggers? I don't see it declared as an arg but idk much about pg funcs

Reply:
Yea NEW is always there for insert triggers